pkg/diagnose/firewall.go

```diff
@@ -45,6 +45,9 @@ const (
 
 const (
 	clientSourcePort = "9898"
+	loadBalancerName = "submariner-gateway"
+	encapsPortName   = "cable-encaps"
```

Comment:
As this is mainly used only as part of log message, lets use "tunnel port" which is more readable.

Reply:
Not sure I understand this comment, these constants are used to retrieve LB port details and not just for logging, port names defined in [operator](https://github.com/submariner-io/submariner-operator/blob/devel/controllers/submariner/loadbalancer_resources.go#L67-#L73)

After we'll merge this PR, I think we should have follow-up PRs, one in the operator (to expose LB constants) and one in subctl to use LB constants from the operator.

Reply:
Yeah, I see that you are also comparing it in the API getLbNodePort(), its fine then.